src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -108,6 +106,18 @@ export default class ReplicationControllerDetailController {
      * @export {boolean}
      */
     this.eventsOrder = DOWNWARDS;
+
+    /**
+     * If true then sidebar will be displayed, otherwise it won't.
+     * @export {boolean}
+     */
+    this.displaySidebar = this.mdMedia('gt-sm');
+
+    /**
+     * Binding resize listener, which switches displaySidebar boolean variable.
+     */
+    angular.element(this.window)
```

Comment:
It would at least double number of lines in the code, so is it really needed? I mean, that I would rather keep it simple. Moving `md-tab` into my new directive is not possible, because it's used in two places and one of them is sidebar. We would need to create new directive for it I think, if I understood correctly.

In your case you had pretty much logic, and here I'm using only `mdMedia`.